.tool-versions

```diff
@@ -1 +1,2 @@
 nodejs 18.17.1
+pnpm 8.6.12
```

Comment:
Does this work? I had trouble getting the pnpm plugin working.

I'll check out the branch and test it out.

Reply:
@mtuchi can we take this out of here and open it up in a new PR?

Looking at the docs we need some better install instructions (ie, add links to the node and pnpm asdf plugins)

It's also a big-ish change and I'd like it merged as a clean PR for the sake of auditing.

Reply:
Certainly